<!DOCTYPE html>
<html>

<head>
  <meta charset="utf-8">
  <title></title>
  <meta name="author" content="">
  <meta name="description" content="">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <link href="style.css" rel="stylesheet">
</head>

<body>
  <div class="layout">


    <a href="https://msbradshaw.github.io/" class="headerItem">About Michael</a>
    <a href="https://msbradshaw.github.io/publications.html" class="headerItem">Publications</a>
    <a href="https://msbradshaw.github.io/experience.html" class="headerItem">CV</a>

    <div class="imgwrap">
      <img src="michael.jpg">
    </div>

    <div class="info">
      <h1>Michael Bradshaw</h1>
      <p>PhD Student</p>
      <p>Computer Science - Computational Genomics</p>
      <p>University of Colorado Boulder</p>
      <h2>Contact</h2>
      <p>Email: michaelscottbradshaw AT gmail.com</p>
    </div>

    <div class="text">
      <p>I am a soon-to-be newly minted PhD in Computer Science, seeking a bioinformatics-related research role. My dissertation work has been in direct collaboration with and support of the Precision Diagnostics team at Childrenâ€™s Hospital of Colorado. Several tools and programs I developed have been in their clinical diagnostics pipelines for 3+ years, including a quality control report for small variants calling in whole exome data, and SeeNV - a tool for visualizing and helping clinicians curate copy-number-variations. Additionally, numerous parameters and tool choices were directly informed by my work.</p>

      <p>Much of my research has focused on biological knowledge graphs and algorithms for clustering, and creating embedding spaces for various machine-learning tasks. I demonstrated solutions for breaking the cyclical nature of biased study design, non-representative databases, and biased predictive models by carefully selecting and curating the underlying knowledge graph used in training. Additionally, by employing historical data updates from widely used ontologies such as HPO and STRING, I built a tool to predict small clusters of genes and phenotypes enriched with latent connections, generating promising hypotheses for 48 undiagnosed patient profiles.</p>

      <p>Prior to graduate school, I interned at the Max Planck Institute for Chemical Ecology for a year and a half. My project there involved quantitative trait loci (QTL) mapping of plant microbiome data and investigating the genetic control of microbiome recruitment.</p>

      <p>Before that, I worked as a web developer for several years and led a team of 7 to build and maintain a suite of websites using PHP, jQuery, and Vue.js. I collaborated closely with content creators and graphic designers to produce intuitive and sleek websites. During this role, I also redesigned our new hire training plan and overhauled our processes with Git to test and deploy new features, removing redundant steps and standardizing developers' testing environments to match our production environment using Docker.</p>

      <p>I enjoy tackling difficult problems and find joy in creating solutions. In my spare time, I like to be in the mountains with my partner, skiing, climbing, running, or some combination of the three.</p>

    </div>

  </div>
</body>
</html>
